diff --git a/setup.py b/setup.py
index ecf3c47..3ec2b3b 100644
--- a/setup.py
+++ b/setup.py
@@ -27,7 +27,7 @@ def main():
             "frida >= 17.2.8, < 18.0.0",
             "prompt-toolkit >= 2.0.0, < 4.0.0",
             "pygments >= 2.0.2, < 3.0.0",
-            "websockets >= 13.0.0, < 14.0.0",
+            "websockets >= 13.0.0, < 16.0.0",
         ],
         license="wxWindows Library Licence, Version 3.1",
         zip_safe=False,
